src/components/Footer.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import { styled } from '@material-ui/styles';
+
+import menuItems from '../services/MenuItems';
+import { getPageUrl } from '../Routes';
+import { spacing, fontSizes } from './styledComponents';
+import logo from '../assets/logo/logo.orange.png';
+import iconFacebook from '../assets/icons/icon.facebook.jpg';
+import iconLinkedIn from '../assets/icons/icon.linkedIn.jpg';
+import iconTwitter from '../assets/icons/icon.twitter.jpg';
+
+
+const socialMedias = [
```

Comment:
`s/socialMedias/socialMediaIcons/`

Reply:
You want me to rename the variable ?

Reply:
Ya, sorry! I was just using sed-style shorthand.